### ISSUE TITLE
[BUG] Close animation doesn't work correctly for phone and chrome

### ISSUE THREAD

**Issue by gfandrea** [Deletescape-Media/Lawnchair#3984]
### Describe the bug

when I close the phone app the closing animation doesn't finish on the icon, but at the bottom left of the screen, same thing with chrome, but the animation end at the bottom right; Doesn't matter the position of the icon on the home screen.
I havent found other app that behave like that.
Both the phone and chrome from the work profile works correctly.

### Steps to reproduce

1. open an app from an icon in the home screen
2. Close it with a swipe

### Expected behavior

the icon should animate going back to the original position

### Device information

- Device: Google Pixel 8
- OS: Android 14 qpr2 beta
- App version: Debug 14 dev 336

**Comment by MrSluffy** [comment]
Please provide info, screen record and if Lawnchair is set as recent provider

**Comment by gfandrea** [comment]
I don't think that lawnchair is set as the recent provider, bacause if I try to change some settings, nothing happens.

But I have activated the setting "activate the application bar" in the recent setting menu and now I see this:
![Screenshot_20240119-192205.png](https://github.com/LawnchairLauncher/lawnchair/assets/34663993/6e6adf98-97ba-4681-b644-cf9c260080fa)


the phone and chrome app close correctly on the app bar icons.
Looks like the animation goes to the hidden app bar, and not to the home screen icon

**Comment by MrSluffy** [comment]
Setting Lawnchair as recent provider using QuickSwitch, Please check group for more info

**Comment by gfandrea** [comment]
I don't have root, so I can't install quickswitch.
Is it possible to disable completelythe recents setting (and the application bar)?

**Comment by MrSluffy** [comment]
> I don't think that lawnchair is set as the recent provider, bacause if I try to change some settings, nothing happens.
> 
> But I have activated the setting "activate the application bar" in the recent setting menu and now I see this: ![Screenshot_20240119-192205.png](https://private-user-images.githubusercontent.com/34663993/298162907-6e6adf98-97ba-4681-b644-cf9c260080fa.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU3NTU1NjQsIm5iZiI6MTcwNTc1NTI2NCwicGF0aCI6Ii8zNDY2Mzk5My8yOTgxNjI5MDctNmU2YWRmOTgtOTdiYS00NjgxLWI2NDQtY2Y5YzI2MDA4MGZhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTIwVDEyNTQyNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYwMWQxNjFlNjg3ZGUwZDBhYjI5YzY5OWI3YTdhMzBkMmE5NzkzZGFhMjQ1NzIwMzIwNTQ3YTI1NzkwMTQ1YTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.lqIyd3TdxXzXMCYpNV39C3Xu0zZFIWsVrx9uuINRsvY)
> 
> the phone and chrome app close correctly on the app bar icons. Looks like the animation goes to the hidden app bar, and not to the home screen icon

does alignment occurs in system icons too? or only lawnicons?

**Comment by gfandrea** [comment]
Yes, also with system icons

**Comment by SuperDragonXD** [comment]
Closing this in favor of #5651.
